vite-project-kabinet/src/Footer/Footer.jsx

```diff
@@ -0,0 +1,20 @@
+import "./footer.style.css"
+import fpu from "../Footer/fpu_logo2_cierne_1.svg"
+
+export const Footer = () => {
+
+    return (
+        <footer>
+            <div className="fond__container">
+                <div className="fond__text-container">
+                    <p className="fond__content">Časopis z verejných zdrojov podporil</p>
+                    <a className="fond__link" href="#">
+                        <img className="fond__image" src={fpu} alt="Fond na podporu umenia logo" />
+                    </a>
+                    <p className="fond__issn">ISSN 2729-7993</p>
+                    <p className="fond__copyright">Copyright © 2024 KABINET | All rights reserved</p>
```

Comment:
Drobnost, ale zkus si jako JS cvičení nahradit "2024" dynamickým výpisem aktuálního roku. Aby jsi to v lednu (každý rok) nemusela upravovat.